src/components/balances/CurrencySelect/__tests__/useCurrencies.test.ts

```diff
@@ -0,0 +1,56 @@
+import { act, renderHook } from '@testing-library/react'
+import { getFiatCurrencies } from '@safe-global/safe-gateway-typescript-sdk'
+import useCurrencies from '../useCurrencies'
+import { Errors, logError } from '@/services/exceptions'
+
+jest.mock('@safe-global/safe-gateway-typescript-sdk', () => ({
+  getFiatCurrencies: jest.fn(),
+}))
+
+jest.mock('@/services/exceptions', () => {
+  const originalModule = jest.requireActual('@/services/exceptions')
+
+  return {
+    ...originalModule, // this will keep the original exports
+    logError: jest.fn(), // this will override logError with a mock function
+  }
+})
+
+describe.only('useCurrencies', () => {
```

Comment:
```suggestion
describe('useCurrencies', () => {
```

I think you left this here by mistake

Reply:
ooops. Thanks for looking.

Reply:
Ok, added a commit with eslint rule that would help us catch those in the future:
https://github.com/safe-global/safe-wallet-web/pull/3064/commits/66d078650861c11609985cd0ee967d9c43e80f1b